open-metadata-implementation/user-interfaces/presentation-server/cra-server/index.js

```diff
@@ -55,11 +55,8 @@ if (env === 'development') {
   // organize routes in another file
   app.use('/', router);
 } else {
-  // use Web Server routing for production build
-
-  // app.use(express.static(`${__dirname}/client/build`));
+  app.use('/', router);
```

Comment:
I do not understand why " app.use('/', router);" is in the if and the else block. If it is needed in both then please move it to before the if.

Reply:
Good catch - I originally thought it needed only to be in development, then realized it was needed in both but did not fix the structure. Just fixed this and pushed now.